docs/csharp/how-to/parse-strings-using-split.md

```diff
@@ -85,12 +85,14 @@ You can use AI tools, such as GitHub Copilot, to generate code to split strings
 The following text shows an example prompt for Copilot Chat:
 
 ```copilot-prompt
-Generate C# code to use Split.String to split a string into substrings.
-Input string is "You win some. You lose some." Delimiters are space and period.
-Provide example output.
+Generate C# code to split a CSV-style input string containing values, where entries may be separated by commas or semicolons and may contain extra whitespace.
+Remove any empty values and trim whitespace from the results.
+Show example output for the string: "  apples, oranges ; bananas, ,  ;pears".
```

Comment:
This is an opinion, but I like the earlier example better, possibly to split a paragraph into distinct sentences using sentence ending punctuation for delimiters:  `.`, `?`, and `!`, and trimming additional whitespace.

Reply:
Discussed with Bill. Keeping the new example but clarifying the scenario better.